etc/env/configs/dev.toml

```diff
@@ -1,3 +1,5 @@
 __imports__ = [ "base", "l1-inits/.init.env", "l2-inits/dev.init.env" ]
 
 ETH_SENDER_SENDER_PUBDATA_SENDING_MODE = "Blobs"
+CHAIN_STATE_KEEPER_BOOTLOADER_HASH="0x010008e57fd7c90fe49f2981d9f2ce8aceeeddf73441204ba0efba6091682a8e"
+CONTRACTS_GENESIS_BATCH_COMMITMENT="0x0d4d2c120413af414cf3c9054b555b5f00928ec1b2ebbdb8d040ac226546989e"
```

Comment:
Do you commit these 2 line accidentally?

Reply:
yes, removed